aries-site/src/pages/templates/global-banner-notifications.mdx

```diff
@@ -12,6 +12,15 @@ import {
   BannerNotificationDiagram,
 } from '../../examples';
 
+<Button
+  icon={<Figma color="plain" />}
+  label="Open in Figma"
+  secondary
+  alignSelf="start"
+  href="https://www.figma.com/file/FGvOd2htjrChVVS2oz73ds/HPE-Notifications?node-id=3597%3A16611"
```

Comment:
Shouldn't we add this as a `figma` prop to the first example instead? That has been our typical convention.

Reply:
I can add it here as well @KennyAtHPE would that be fine? was there any particular reason you wanted the button separate or can we add it to the example

Reply:
Hm, I'm thinking that having it at the top similar to the page layouts will be the easiest for users to see it, but if standard practice of having it on the example works then that works too

Reply:
I added it to the example how we have in our other example. I agree since this is a standard practice also see how it can be missed. I'm wondering if this was part of the user study if they were able to find the figma link for each page something to think about.